R/assocTestAggregate.R

```diff
@@ -10,7 +10,7 @@ setMethod("assocTestAggregate",
           function(gdsobj, null.model, AF.max=1,
                    weight.beta=c(1,1), weight.user=NULL,
                    test=c("Burden", "SKAT", "fastSKAT", "SMMAT", "SKATO"),
-                   burden.test=c("Score", "Wald"),
```

Comment:
Are we leaving the burden.test argument here in anticipation of adding BinomiRare and CMP? Or should those go in the "test" argument instead? https://github.com/UW-GAC/GENESIS/pull/32#discussion_r326760137